templates/ContentGenerator/Instructor/PGProblemEditor/save_form.html.ep

```diff
@@ -16,16 +16,18 @@
 	% my $backupMax = $c->ce->{options}{editorNumberOfBackups};
 	% if ($backupMax > 0) {
 		<div class="form-check mb-2">
-			<%= check_box backupFile => 1, id => 'backupFile', class => 'form-check-input', checked => undef =%>
+			% param('backupFile', 1) unless defined param('backupFile');
+			<%= check_box backupFile => 1, id => 'backupFile', class => 'form-check-input' =%>
+			<%= hidden_field backupFile => 0 =%>
```

Comment:
Note that this is the standard construct for "checked by default, but preserve what the user does on form submission".